graphql/resolvers/tasks.js

```diff
@@ -0,0 +1,37 @@
+const Task = require("../../models/task");
+const moment = require('moment');
+
+module.exports = {
+  tasks: async (args) => {
+    try {
+      let tasks = await Task.find({ userId: args.userId }).populate(
+        "assignedBy"
+      );
+      return tasks.map((result) => {
+        return { ...result._doc,date: moment(result._doc.date).format('ll') };
+      });
+    } catch (err) {
+      throw err;
+    }
+  },
+  createTasks: async (args, req) => {
+    if (!req.isAuth) {
```

Comment:
as you have already made a middleware for auth so don't add any auth check here.

Reply:
I'm checking the middleware on specific requests because I haven't setup this middleware for all api's. hence, using this if statements where this auth check is required, other APIs don't have these. Let me know if there's a better approach